### PR TITLE
refactor: Split functionality into multiple files

### LINKED ISSUE #33: refactor: Split functionality into multiple files

### PR DESCRIPTION
**Pull request by joe-sacco**

### CODE REVIEW

File: src/send_data.py [hunk 1]
```diff
@@ -1,71 +1,43 @@
...
+            
+        except RequestException as e:
+            print("Request failed: ", e)
+            pass
+
+        print('Successfully send the data to server')
 
     except Exception as e:
-        raise e
+        print("Some failure occurred when monitoring sensor")
```
Comment (Takashicc) [inline src/send_data.py]: Print `e` too.
When we faced this `print` we cannot know exactly where did the error come from.

File: src/send_data.py [hunk 2]
```diff
@@ -1,71 +1,43 @@
...
-        e: Failed to get data from sensor
+# URL = os.getenv("SERVER_URL")
+# DEVICE_ID = '1'
 
-    Returns:
-        dict: Dictionary of sensor data
-    """
-    
+def send_data(data, URL, DEVICE_ID):
     try:
```
Comment (Takashicc) [inline src/send_data.py]: In this scope, `URL` and `DEVICE_ID` is not a global variable.
Make it lowercase.

File: src/send_data_bak.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
+from sensor_temp import get_temp
```
Comment (Takashicc) [inline src/send_data_bak.py]: Don't add backup files in PR.
If you want to keep it in your local, Add `*_bak.py` in `.gitignore`

File: src/sensor_data_dict.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+def create(alarm_status, button_home_status):
+    """Create data from multiple sensors
+
+    Raises:
+        e: Failed to get data from sensor
+
+    Returns:
+        dict: Dictionary of sensor data
+    """
```
Comment (Takashicc) [inline src/sensor_data_dict.py:16]: Missing `Args:`.
https://google.github.io/styleguide/pyguide.html#383-functions-and-methods

**Review by Takashicc — CHANGES_REQUESTED** [review]

File: src/send_data.py (hunk 1 above)
Comment (joe-sacco) [reply src/send_data.py:30]: I think this was something you added.   Would you like us to delete the print?

File: src/send_data_bak.py (hunk 3 above)
Comment (joe-sacco) [reply src/send_data_bak.py]: Ok, but then we won't have the backup saved anywhere if something happens to our computer.

File: src/sensor_data_dict.py (hunk 4 above)
Comment (joe-sacco) [reply src/sensor_data_dict.py:16]: You add this section.   Please let us know exactly how you'd like us to type it.

File: src/send_data.py (hunk 2 above)
Comment (joe-sacco) [reply src/send_data.py]: Done

File: src/send_data.py (hunk 1 above)
Comment (joe-sacco) [reply src/send_data.py]: Done

File: src/send_data_bak.py (hunk 3 above)
Comment (joe-sacco) [reply src/send_data_bak.py]: Added to gitignore but probably too late since we already committed it.  It still shows up.

Comment (Takashicc) [reply src/send_data_bak.py]: We use `git`. `git` will track all the files that you committed.

**Review by Takashicc — APPROVED** [review]

**Review by Takashicc — APPROVED** [review]

**Pull request merged**